scripts/ngless-cwl/setup.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python
+
+import setuptools
+from glob import glob
+
+setuptools.setup(
```

Comment:
If you want to be super fancy, you can use https://docs.python.org/2/distutils/setupscript.html#installing-additional-files to install the CWL descriptions into (`sys.prefix/`)  `share/commonwl` so that [they get autodetected by CWL platforms](http://www.commonwl.org/v1.0/CommandLineTool.html#Discovering_CWL_documents_on_a_local_filesystem).